particle/data/particle2008.csv

```diff
@@ -477,13 +477,13 @@ ID,Mass,MassUpper,MassLower,Width,WidthUpper,WidthLower,I,G,P,C,Anti,Charge,Rank
 300553,10579.4,1.2,1.2,20.5,2.5,2.5,0,-1,-1,-1,0,0,0,0,Upsilon(4S),bB,\Upsilon(4S)
 9000111,984.7,1.2,1.2,75,25,25,1,-1,1,1,2,0,0,0,a(0)(980),Maybe non-qQ,a_{0}(980)^{0}
 9000113,1351,30,30,310,40,40,1,-1,-1,1,0,0,0,0,pi(1)(1400),Maybe non-qQ,\pi_{1}(1400)^{0}
-9000115,2090,29,29,620,50,50,1,-1,-1,1,2,0,0,2,pi(2)(2100),(uU-dD)/sqrt(2),\pi_{2}(2100)^{0}
+9000115,2090,29,29,620,50,50,1,-1,-1,1,2,0,0,2,pi(2)(2100),(uU-dD)/sqrt(2),a_{2}(1700)^{0}
```

Comment:
The "name" needs to be changed too

Reply:
Ah, this is a side effect of the way we fix/update PDG information. This file is automatically generated but not really used anywhere. It should probably be removed, actually, as it is bound to always have issues since old and most-up-to-date information. I won't change these spurious things by hand but will see if Henry agrees to just not have them in the package, as it can be / is a source of confusion :S. Thanks for the good eyes.